components/VerifyOptions.yml

```diff
@@ -23,11 +23,15 @@ components:
         domain:
           type: string
           description: The intended domain of validity for the proof. For example website.example
+        returnCredential:
+          type: boolean
```

Comment:
Maybe `boolean` is not a valid keyword and this is why the extra json is appearing

Reply:
Fixed by https://github.com/w3c-ccg/vc-api/pull/366/commits/5fa7f3ae6e840587edeb263aa95a0dcc0c3c6b09